lib/environments/nextcloud.js

```diff
@@ -0,0 +1,11 @@
+"use strict";
+
+module.exports = {
+	globals: {
+		$: 'readonly',
```

Comment:
I don't think this is documented officially yet but we are trying really hard to get rid of these global libraries as they cause update issues. This is one of the reasons why we can't have the latest jquery in server right now as it breaks so many app :cry:

Reply:
okay, I've removed it